Network/Public/Include/Network/CombiningHandlers.hpp

```diff
@@ -14,36 +14,36 @@ namespace Network
 		public:
 		/**
 		* @brief Method for serialization, compression and encryption of outcoming messages.
-		* @param message    - buffer that contains data that should be serialized, compressed and encrypted.
-		* @param bodyBuffer - buffer that will contain serialized, compressed and encrypted body.
+		* @param source      - buffer that contains data that should be serialized, compressed and encrypted.
+		* @param destination - buffer that will contain serialized, compressed and encrypted body.
 		*/
-		MessageProcessingState operator()(const Message& message,yas::shared_buffer& bodyBuffer) const
+		static MessageProcessingState handleOutcomingMessage(const Message& source,yas::shared_buffer& destination)
 		{
-			auto serializationHandler = std::make_unique<SerializationHandler>();
-			auto compressionHandler   = std::make_unique<CompressionHandler>();
-			auto encryptionHandler    = std::make_unique<EncryptionHandler>();
+			static auto serializationHandler = std::make_unique<SerializationHandler>();
```

Comment:
The same handler for all messages?

Reply:
I mean static qualifier. It make the same memory for all call every time. Is that what you trying to reach?

Reply:
Yes, to avoid reallocating memory on subsequent function calls.